test.py

```diff
@@ -9,8 +11,17 @@
 
 print oc.toRadString()
 
+
+genday = Gendaymtx(v=True,h=True)
+genday.inputFiles =[r'C:\Ladybug\annual\annualSimulation\State_College__Penn_State___Su_PA_USA.wea'] #This path needs to be present physically !
```

Comment:
Since gendaymtx accepts wea files as a very specific file type I would say it's better to have a property for wea files (```genday.wea = "filepath"```)